.github/workflows/main.yml

```diff
@@ -153,5 +149,26 @@ jobs:
       # ** End **
 
       # ** Setup old databases and compare (test `down`) **
-      # TBD
+      - name: Setup old database
+        working-directory: ./alteration
+        run: |
+          cd packages/cli
+          pnpm start db seed
+        env:
+          DB_URL: postgres://postgres:postgres@localhost:5432/old
+
+      - name: Revert fresh database to old
+        working-directory: ./fresh
+        run: pnpm cli db alt r v1.0.0-beta.11
+        env:
+          DB_URL: postgres://postgres:postgres@localhost:5432/fresh
+
+      - name: Compare manifests
+        working-directory: ./fresh
+        run: node .scripts/compare-database.js fresh old
       # ** End **
+
+      - name: Check alteration sequence
+        continue-on-error: true # Temporarily skip this as we are fixing previous down scripts
```

Comment:
Should we remove this now, since it's more like a debugging flag?

Reply:
remove in #2976